internal/engine/build_and_deployer.go

```diff
@@ -74,8 +74,9 @@ func (composite *CompositeBuildAndDeployer) BuildAndDeploy(ctx context.Context,
 		}
 
 		if redirectErr, ok := err.(buildcontrol.RedirectToNextBuilder); ok {
-			s := fmt.Sprintf("falling back to next update method because: %v\n", err)
-			logger.Get(ctx).Write(redirectErr.Level, []byte(s))
+			l := logger.Get(ctx).WithFields(logger.Fields{logger.FieldNameBuildEvent: "1"})
```

Comment:
can we make this more descriptive than "0" / "1"? maybe "init" and "fallback"?

Reply:
Oh geez, for some reason I got stuck in an earlier era of programming history and thought enums could only be integers. But clearly that's not even an integer, it's a string... 🙄